src/toolkits/object_detection/one_shot_object_detection/CMakeLists.txt

```diff
@@ -1,14 +1,7 @@
 project(unity_toolkits)
 
-make_library(unity_one_shot_object_detection OBJECT
-  SOURCES
-    class_registrations.cpp
-    util/parameter_sampler.cpp
-    util/superposition.cpp
-    one_shot_object_detector.cpp
-  REQUIRES
-    image_io
-    random
-    table_printer
-    unity_core
-)
+    make_library(unity_one_shot_object_detection OBJECT SOURCES
```

Comment:
Let's revert the changes in this file - I'm not sure clang-format got it right.

Reply:
Yes, in fact, this no longer builds...

Reply:
Fixed.